js/main.js

```diff
@@ -27,6 +30,26 @@ Site = {
   },
 };
 
+Site.Press = {
```

Comment:
What about other galleries in other parts of the site? Can this functionality be re-used as it is?
Would be convenient to use a more generic name? ex. `Site.Sliders`